site/zenodo_rdm/metrics/ext.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""ZenodoRDM Metrics module."""
+
+from flask import current_app
+
+from zenodo_rdm.metrics import config
+
+
+class ZenodoMetrics(object):
+    """Zenodo frontpage extension."""
+
+    def __init__(self, app=None):
+        """Extension initialization."""
+        if app:
+            self.init_app(app)
+
+    @staticmethod
+    def init_config(app):
+        """Initialize configuration."""
+        for k in dir(config):
+            if k.startswith("METRICS"):
```

Comment:
```suggestion
            if k.startswith("METRICS_"):
```

A mistake from the original module as well I guess ;)